src/TodoList.js

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import TodoListItem from './TodoListItem';
 
-const todoList = [
-    {
-        id: 1,
-        title: 'Complete assignment',
-    },
-    {
-        id: 2,
-        title: 'Complete mindset assignment',
-    },
-    {
-        id: 3,
-        title: 'Submit assignment',
-    }
-];
+// const todoList = [
+//     {
+//         id: 1,
+//         title: 'Complete assignment',
+//     },
+//     {
+//         id: 2,
+//         title: 'Complete mindset assignment',
+//     },
+//     {
+//         id: 3,
+//         title: 'Submit assignment',
+//     }
+// ];
 
-let TodoList = () => {
+let TodoList = ({ todoList }) => {
     return (
         <ul>
             {todoList.map(function (item) {
-                return (<TodoListItem key={item.id} todo={item} />);
+                return (<TodoListItem key={item} todo={item} />);
```

Comment:
To fix this bug I recommend making `key={item}` unique, for example, use the `id` of the `item`

<img width="1039" alt="Screen Shot 2023-11-04 at 11 08 28 AM" src="https://github.com/AlmiraKoshkina/react-todo/assets/11164899/43c9889d-cc7e-4d79-ad20-98192d625594">